system/modules/core/elements/ContentDownloads.php

```diff
@@ -21,7 +21,7 @@ class ContentDownloads extends \ContentElement
 
 	/**
 	 * Files object
-	 * @var \FilesModel
```

Comment:
You could actually use `@var \FilesModel[]` here and then `foreach` over the values. Then the IDE would already know the type of each item in the foreach-loop.